### PR TITLE
feat: Adds webpack support for azure plugin

### PR DESCRIPTION
**Pull request by wbreza**
When azure serverless project is deploy with webpack we need to ensure that the required files for azure functions are part of the deployment package.

- Copies host.json & function.json files for each serverless function.

### CODE REVIEW

File: src/plugins/login/loginPlugin.test.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ describe("Login Plugin", () => {
     const options = MockFactory.createTestServerlessOptions();
```
Comment (mydiemho) [inline src/plugins/login/loginPlugin.test.ts]: this login order change has nothing to do with webpack, I would put this in a different pr

File: src/plugins/login/loginPlugin.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export class AzureLoginPlugin {
     this.provider = (this.serverless.getProvider("azure") as any) as AzureProvider;
 
     this.hooks = {
```
Comment (mydiemho) [inline src/plugins/login/loginPlugin.ts]: same as above

File: src/plugins/package/azurePackage.ts [hunk 3]
```diff
@@ -1,54 +1,44 @@
...
+import { PackageService } from "../../services/packageService";
 
 export class AzurePackage {
+  private bindingsCreated: boolean = false;
+  private packageService: PackageService;
   public provider: AzureProvider;
   public hooks: { [eventName: string]: Promise<any> };
 
-  public constructor(private serverless: Serverless, private options: Serverless.Options) {
+  public constructor(private serverless: Serverless) {
```
Comment (tbarlow12) [inline src/plugins/package/azurePackage.ts]: Reason for no options?

Comment (wbreza) [reply src/plugins/package/azurePackage.ts]: The plugin doesn't use any options right now - so removed it.  If it starts to take options in the future I'm fine bringing it back.

File: src/plugins/login/loginPlugin.test.ts (hunk 1 above)
Comment (wbreza) [reply src/plugins/login/loginPlugin.test.ts]: Reverted login changes.

File: src/plugins/login/loginPlugin.ts (hunk 2 above)
Comment (wbreza) [reply src/plugins/login/loginPlugin.ts]: Reverted login changes.
